docs/articles/documentation/reference/test-api/testcontroller/presskey.md

```diff
@@ -78,6 +78,10 @@ Shortcut        | Elements
 
 > The `'backspace'`, `'delete'`, `'left'` and `'right'` key presses in `contentEditable` elements are processed only when text is selected.
 
+<!-- -->
+
+> Key combinations of `Ctrl` + `+` / `Ctrl` + `-` don't zoom the page in our out.
```

Comment:
I think it's better to add a row to the table below, similarly as we did with `Ctrl+C`/`Ctrl+V`.

Reply:
Done